config/scheduledfeed.go

```diff
@@ -135,9 +143,12 @@ func (pc PublisherConfig) ToPublisher(ctx context.Context) (publisher.Publisher,
 	case stdout.PublisherType:
 		return stdout.New(), nil
 	default:
-		err = fmt.Errorf("%w : %v", errUnknownPub, pc.Type)
+		return nil, fmt.Errorf("%w : %v", errUnknownPub, pc.Type)
 	}
-	return nil, err
+}
+
+func (fc FeedConfig) ToFeed() (feeds.ScheduledFeed, error) {
```

Comment:
Is there a need to have the public constructor behind this func?

Reply:
I think this remains after some shifting around of code, the wrapped function was used elsewhere without a FeedConfig, but this is no longer needed so I'll simplify this down to just `ToFeed()`.